### DISCUSSION TITLE
MIT 6.824 Lab3 KVRaft 笔记 | XX’s Blog

### DISCUSSION THREAD

**Discussion by xxchan** [xxchan/xxchan.github.io#18] [Announcements]
https://xxchan.github.io/cs/2021/02/28/6.824-kvraft.html 

 Just some random thoughts of mine.

**Comment by izackwu** [comment]
我当时写的时候也是卡在了 `TestSnapshotSize3B`，原因和解决办法就是你说的。但是明明在 Lab 2 的时候，说好了心跳包的频率不能过高呢：

> The tester requires that the leader send heartbeat RPCs no more than ten times per second.

所以我也被困在这里好久……😂
